tests/unit/replicator_tests.py

```diff
@@ -86,8 +86,9 @@ def test_constructor_failure(self):
         """
         repl = None
         try:
-            self.client.disconnect()
-            repl = Replicator(self.client)
+            client = copy.deepcopy(self.client)
+            client.disconnect()
+            repl = Replicator(client)
```

Comment:
is this `client` copy necessary anymore?  If so, I'd prefer it were just a clean contruction of a new client rather than a deepcopy of the `self.client`.